### ISSUE TITLE
Disabled refactor code action isn't accessible 

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode#86021]
Testing https://github.com/microsoft/vscode/issues/85993

Given this refactor context menu:
![image](https://user-images.githubusercontent.com/38270282/70040839-73429600-15bc-11ea-8570-ba19601aa1ff.png)

I cannot use keyboard navigation to get to the disabled menu item. When I have a screen reader on, I can't get it to read the disabled menu item (either by keyboard or by mouse).

**Comment by mjbvz** [comment]
@sbatten Adding you since we are just using VS Code's context menu service

**Comment by sbatten** [comment]
interesting, didn't think about this too much in the past. interestingly, i see that many native Windows context menus let you "select" disabled items to read with the screen reader. The "native" electron menus however do no allow this, what's happening on macOS?

**Comment by isidorn** [comment]
That is fine I think since  disabled entries on native mac menu behave the same way.
Thus closing
